superset/db_engine_specs.py

```diff
@@ -1526,6 +1590,10 @@ def where_latest_partition(
                     return qry.where(Column(col_name) == value)
         return False
 
+    @classmethod
+    def _get_fields(cls, cols: List[dict]) -> List[ColumnClause]:
+        return BaseEngineSpec._get_fields(cols)
+
```

Comment:
Why is this needed? Isn't the class derived from `BaseEngineSpec`?

Reply:
The class is derived from PrestoEngineSpec, which is then derived from BaseEngineSpec.

Reply:
Ah, got it! :)